deployments/with-creds/hush-house/requirements.lock

```diff
@@ -1,6 +0,0 @@
-dependencies:
```

Comment:
Although this was indeed referencing an old version, I think it's still important to keep lock files as they list the digests and creation timestamps of our direct dependencies 🤔

Reply:
We undid this change